### DISCUSSION TITLE
rutorrent not downloading..

### DISCUSSION THREAD

**Discussion by gerberanub** [crazy-max/docker-rtorrent-rutorrent#309] [General]
torrent added but not downloading.. no response.

1704263242 W 1092DBBBBC14E9C99A9B127EB7C541C2CC0BE442->tracker_list: could find matching tracker protocol (url:wss://tracker.openwebtorrent.com:443/announce)
1704263242 N rtorrent main: Starting thread.
1704263242 N rtorrent scgi: Starting thread.

please help me..

**Comment by thomasmarteau** [comment]
Did you try to resolve any dns stuff within the container. Maybe your firewall is the culprit.
